Lib/test/test_tools/test_reindent.py

```diff
@@ -23,6 +23,13 @@ def test_help(self):
         self.assertEqual(out, b'')
         self.assertGreater(err, b'')
 
+    def test_reindent_file_with_bad_encoding(self):
+        bad_coding_path = os.path.join(
+            os.path.dirname(os.path.realpath(__file__)), '../bad_coding.py')
+        rc, out, err = assert_python_ok(self.script, '-r', bad_coding_path)
+        self.assertEqual(out, b'')
+        self.assertGreater(err, b'')
```

Comment:
I think `assertNotEqual()` is more appropriate here. It produces more detailed report for strings and perhaps will produce more detailed report for bytes in future.